crates/typos-dict/assets/words.csv

```diff
@@ -687,7 +687,6 @@ accredidation,accreditation
 accreditied,accredited
 accreditted,accredited
 accress,access
-accreting,accrediting
```

Comment:
I worry about this being added again in the future.  Could you add this to https://github.com/crate-ci/typos/blob/master/crates/typos-dict/assets/allowed.csv so we prevent this from happening again?

Reply:
Thanks for pointing this out. Now added to `allowed.csv`.